.github/workflows/master-to-main.yml

```diff
@@ -0,0 +1,77 @@
+name: Convert the master branch to the main branch and set up a monorepo
+
+on:
+  workflow_dispatch:
+
+jobs:
+  master-to-main:
+    if: github.triggering_actor == "nickvigilante"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.5.0
+        with:
+          ref: master-2
+          path: source
+      - uses: actions/checkout@v3.5.0
+        with:
+          ref: main-2
```

Comment:
Yep, I'm just gonna be dropping and recreating `master-2` as a copy of the latest `master`. The reason I did that was because I wasn't sure about how nicely the add and commit GH action was gonna play with our repo, and I don't want it accidentally committing to `master`.

Reply:
💯

Reply:
Although, I'm glad you asked this, because I realized that I can specify a `cwd` for that action, so it knows it must use the `target` directory. Completely forgot about this.